packages/lesswrong/components/events/modules/VirtualProgramCard.tsx

```diff
@@ -104,30 +104,17 @@ const VirtualProgramCard = ({program, classes}: {
 }) => {
   const { captureEvent } = useTracking()
   
-  // find the next deadline for applying to the Intro VP, which is the last Sunday of every month
-  let deadline = moment()
-  // Iterate through 5 Sundays to find the one we want
-  for (const sunday of _.range(5).map(x => moment().day(0).add(x, 'week'))) {
-    const nextSunday = moment(sunday).add(1, 'week')
-    // needs to be in the future, and needs to be the last Sunday of the month
-    if (sunday.isSameOrAfter(moment(), 'day') && nextSunday.month() !== sunday.month()) {
-      deadline = sunday
-      break
-    }
-  }
-  // this Dec, the dates are different
-  // TODO: remove once 2023 arrives
-  const dec11 = moment("12-11-2022", "MM-DD-YYYY") // deadline for Jan-Feb cohort (start date is Jan 9)
-  const jan1 = moment("01-01-2023", "MM-DD-YYYY")
-  const jan29 = moment("01-29-2023", "MM-DD-YYYY") // deadline for Feb-Mar cohort? (should be normal start date logic)
-  if (moment().isSameOrBefore(dec11)) {
-    deadline = dec11
-  } else if (moment().isSameOrBefore(jan1)) {
-    deadline = jan29
+  // Find the next deadline for applying to the Intro VP, which is usually the 4th Sunday of every month
+  // (though it will sometimes move to the 5th Sunday - this is not accounted for in the code).
+  // This defaults to the Sunday in the week of the 28th day of this month.
+  let deadline = moment().date(28).day(0)
+  // If that Sunday is in the past, use next month's 4th Sunday.
+  if (deadline.isBefore(moment())) {
+    deadline = moment().add(1, 'months').date(28).day(0)
```

Comment:
What happens if:
* Today is the 31st
* I add one month
* Next month has <31 days
Does it round down to the max day of the next month, or roll over to the following month?

Reply:
Ah good question, I didn't test that case. According to the [docs](https://momentjs.com/docs/#/manipulating/):

> If the day of the month on the original date is greater than the number of days in the final month, the day of the month will change to the last day in the final month.

Which is what we want, so it should be good.